packages/fresha-api/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
`@fresha/fresha-api` is code moved from `@fresha/fresha-spa` without changes?

Reply:
@mgrabowski84  Mostly. I copied code from `@fresha/partners-dml` and tried to retain as much as possible of it.